command/connect/envoy/bootstrap_config.go

```diff
@@ -557,6 +557,7 @@ func (c *BootstrapConfig) generateListenerConfig(args *BootstrapTplArgs, bindAdd
 
 	clusterJSON := `{
 		"name": "` + selfAdminName + `",
+		"ignore_health_on_host_removal": false,
```

Comment:
This is a default setting only set to force envoy to notice this should be definitely v3.